### PR TITLE
Improve coverage of mkt/extensions to 100%

### PR DESCRIPTION
**Pull request by diox**

### CODE REVIEW

File: mkt/extensions/tests/test_validation.py [hunk 1]
```diff
@@ -163,7 +209,7 @@ def test_author_missing_valid(self):
 
     def test_author_too_long(self):
         with self.assertRaises(ParseError):
-            self.validator.validate_name({'name': u'ŷ' * 129})
+            self.validator.validate_author({'author': u'ŷ' * 129})
```
Comment (diox) [inline mkt/extensions/tests/test_validation.py]: yeah...

File: mkt/extensions/validation.py [hunk 2]
```diff
@@ -111,7 +111,7 @@ def validate_json(self, contents):
         try:
             # We support only UTF-8 encoded manifests.
             decoded_data = smart_unicode(strip_bom(contents))
-        except (ValueError, UnicodeDecodeError):
```
Comment (diox) [inline mkt/extensions/validation.py]: `ValueError` was never thrown, I must have copy/pasted it from somewhere else.

File: mkt/extensions/views.py [hunk 3]
```diff
@@ -54,10 +54,6 @@ def create(self, request, *args, **kwargs):
         if not upload_pk:
             raise exceptions.ParseError(_('No validation_id specified.'))
 
-        if not request.user.is_authenticated():
```
Comment (diox) [inline mkt/extensions/views.py]: This wasn't needed, all the views implementing this mixin use permissions. There are tests covering this.

**Comment by ngokevin** [comment]
r+, although r- on using your time off on code quality.

**Comment by diox** [comment]
But playing with coverage was fun ! :)

**Pull request merged**